payments-core/src/main/java/com/stripe/android/googlepaylauncher/GooglePayLauncherModule.kt

```diff
@@ -18,6 +21,8 @@ class GooglePayLauncherModule {
         DefaultGooglePayRepository(
             appContext,
             environment,
+            GooglePayJsonFactory.BillingAddressParameters(),
+            true,
```

Comment:
So for Payment Sheet we'll always require an existing Payment Method? I know this is the existing behavior, but thinking that we could set to false so Google Pay will still be an option and the user can add the Payment Method during checkout.

Reply:
I think we should give developer more control, so he/she can manage the flow

Reply:
I did ask this question.  The plan for now is to match iOS.